jenkins/ci.suse.de/templates/cloud-update-ci-trigger-template.yaml

```diff
@@ -0,0 +1,19 @@
+- job-template:
+    name: 'cloud-update-ci-{instance}-trigger'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H */3 * * *'
```

Comment:
@dirkmueller 
That is the main idea behind it. To remove all manual changes.
What about doing it once per day at 01:00 in the morning?